app/src/main/java/corp/kairos/adamastor/ContextList/AppViewHolder.java

```diff
@@ -0,0 +1,38 @@
+package corp.kairos.adamastor.ContextList;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.v4.content.res.ResourcesCompat;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import corp.kairos.adamastor.R;
+
+public class AppViewHolder extends RecyclerView.ViewHolder {
+    public View rootView;
+    public TextView labelView;
+    public ImageView iconView;
+    public String packageName;
+
+    public AppViewHolder(View itemView, Context appContext) {
+        super(itemView);
+
+        rootView = itemView;
+        iconView = (ImageView) itemView.findViewById(R.id.app_image);
+        labelView = (TextView) itemView.findViewById(R.id.app_text);
+        // TODO: Make design default like this
+        labelView.setTextColor(
```

Comment:
I imagine this cannot be done in the view resouce, right? :smile:

Reply:
Since I'm using the same resource as the all apps, If I change it there, I'll change it in the all apps too, where the background is also white, making the label invisble.

Reply:
:+1: